blue-note/theme.json

```diff
@@ -0,0 +1,224 @@
+{
+	"$schema": "https://schemas.wp.org/wp/6.2/theme.json",
+	"customTemplates": [
+		{
+			"name": "single-no-sidebar",
+			"postTypes": [
+				"page",
+				"post"
+			],
+			"title": "Single, no sidebar"
+		}
+	],
+	"settings": {
+		"appearanceTools": true,
+		"color":{
+			"palette": [
+				{
+					"slug": "base",
+					"name": "Base",
+					"color": "#F5F3EA"
+				},
+				{
+					"slug": "contrast",
+					"name": "Contrast",
+					"color": "#000000"
+				},
+				{
+					"slug": "primary",
+					"name": "Primary",
+					"color": "#000000"
+				},
+				{
+					"slug": "secondary",
+					"name": "Secondary",
+					"color": "#6E6E6E"
+				},
+				{
+					"slug": "tertiary",
+					"name": "Tertiary",
+					"color": "#ADADAD"
+				}
+			],
+			"duotone": [
+				{
+					"slug": "duotone-1",
+					"name": "Duotone 1",
+					"colors": [
+						"#000000",
+						"#C8266A"
+					]
+				},
+				{
+					"slug": "duotone-2",
+					"name": "Duotone 2",
+					"colors": [
+						"#000000",
+						"#3371BD"
+					]
+				},
+				{
+					"slug": "duotone-3",
+					"name": "Duotone 3",
+					"colors": [
+						"#000000",
+						"#92A317"
+					]
+				},
+				{
+					"slug": "duotone-4",
+					"name": "Duotone 4",
+					"colors": [
+						"#000000",
+						"#FCE9A5"
+					]
+				},
+				{
+					"slug": "duotone-5",
+					"name": "Duotone 5",
+					"colors": [
+						"#000000",
+						"#EA5200"
+					]
+				},
+				{
+					"slug": "duotone-6",
+					"name": "Duotone 6",
+					"colors": [
+						"#000000",
+						"#6C4A8F"
+					]
+				},
+				{
+					"slug": "duotone-7",
+					"name": "Duotone 7",
+					"colors": [
+						"#000000",
+						"#FFFFFF"
+					]
+				}
+			]
+		},
+		"layout": {
+			"contentSize": "655px",
+			"wideSize": "874px"
```

Comment:
these values are a bit strange, and I see looking at the figma how it's not clear what they should be. I'd settle with `contentSize` as 874px and maybe add padding to the cases where we want it narrower (or find a middle ground and equalize all of the widths) and have `wideSize` be 1300px